src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/DocAssemblyService.java

```diff
@@ -68,12 +68,31 @@ public DocAssemblyResponse createOrder(CCDCase ccdCase, String authorisation) {
         );
     }
 
+    public DocAssemblyResponse createGeneralLetter(CCDCase ccdCase, String authorisation, String templateId) {
+        UserDetails userDetails = userService.getUserDetails(authorisation);
+
+        logger.info("Doc assembly service: creating request for doc assembly");
+
+        DocAssemblyRequest docAssemblyRequest = DocAssemblyRequest.builder()
+            .templateId(templateId)
+            .outputType(OutputType.PDF)
+            .formPayload(docAssemblyTemplateBodyMapper.generalLetterBody(ccdCase, userDetails))
+            .build();
+
+        logger.info("Doc assembly service: sending request to doc assembly");
```

Comment:
Logging could be more descriptive
```suggestion
        logger.info("Doc assembly service: sending general letter request to doc assembly");
```